app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBListeners.tsx

```diff
@@ -358,6 +378,14 @@ export class ALBListeners extends React.Component<IALBListenersProps, IALBListen
       .catch(() => {});
   };
 
+  private configureRedirect = (action: IListenerAction): void => {
+    ConfigureRedirectConfigModal.show({ config: action.redirectActionConfig })
```

Comment:
Is this a modal inside a modal?

Reply:
Yes

Reply:
This entire dialog needs to be re-designed. We already have a few places we use modals in modals, so just following existing patterns until we figure out these forms better. No point in putting lipstick on a pig we're going to slaughter.